django_redshift_backend/base.py

```diff
@@ -553,6 +555,162 @@ class DatabaseCreation(BasePGDatabaseCreation):
     pass
 
 
+class DatabaseIntrospection(BasePGDatabaseIntrospection):
+    pass
+
+    def get_table_description(self, cursor, table_name):
+        """
+        Return a description of the table with the DB-API cursor.description
+        interface.
+        """
+        # Query the pg_catalog tables as cursor.description does not reliably
+        # return the nullable property and information_schema.columns does not
+        # contain details of materialized views.
+
+        # This function is based on the version from the Django postgres backend
+        # from before support for collations were introduced in Django 3.2
```

Comment:
nits: I'll add a URL to original django source after merging:

```suggestion
        # from before support for collations were introduced in Django 3.2
        # https://github.com/django/django/blob/3.1.14/django/db/backends/
        # postgresql/introspection.py#L66-L94
```